assets/scss/modules/_datatables.scss

```diff
@@ -121,3 +121,66 @@
     cursor: pointer;
   }
 }
+
+.js-datatable-wrapper[action$="client-list"] {
+  th:first-child {
```

Comment:
the .js-datatable-wrapper[action$="client-list"] rule itself no longer exists as a selector in the AOSS client list page.

As no other team has a client-list I will remove the entire rule.